src/openforms/js/components/admin/form_design/variables/prefill/PrefillConfigurationForm.js

```diff
@@ -23,14 +23,16 @@ const PrefillConfigurationForm = ({
   attribute = '',
   identifierRole = 'main',
   // TODO: find a better way to specify this based on the selected plugin
-  options = {
+  options = {},
+  errors,
+}) => {
+  const defaults = {
     objectsApiGroup: '',
-    objecttype: '',
+    objecttypeUuid: '',
     objecttypeVersion: null,
     variablesMapping: [],
-  },
-  errors,
-}) => {
+  };
+  options = {...defaults, ...options};
```

Comment:
I think this needs to be pushed down to `ObjectsAPIFields` and have it set the default values on initial render - you can get the `initialValues` from the Formik state. This generic component should not be polluted with plugin-specific options and we want to keep the relevant code co-located.